modules/roles_profiles/manifests/profiles/chrome.pp

```diff
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Not sure you need an profile here but if you are, call it google_chrome and not just chrome

Reply:
@dividehex I renamed the profile. I want to keep this as profile, so it is explicit in the role if Chrome is installed or not.